webpack/scenes/AlternateContentSources/Create/Steps/AcsUrlPaths.js

```diff
@@ -71,6 +90,15 @@ const AcsUrlPaths = () => {
             onChange={checked => setVerifySSL(checked)}
           />
         </FormGroup>
+        {verifySSL && acsType === 'rhui' &&
+          <>
+            {__('Generate certificates for the chosen repositories')}
+            <ClipboardCopy hoverTip="Copy" clickTip="Copied" variant="inline-compact" isBlock>
+              rhui-manager client cert --name rhui-acs-certs --days 365 --dir /root --repo_label rhui-repo-1,rhui-repo-2
+            </ClipboardCopy>
+            {__('Create content credentials in server with the created SSL certificate and key via Content -> Content Credentials')}
```

Comment:
I think "in the server" would be implied by Content -> Content Credentials
```suggestion
            {__('Create content credentials with the generated SSL certificate and key via Content -> Content Credentials.')}
```